release/staging-build.yml

```diff
@@ -205,3 +205,21 @@ batch:
     depend-on:
     - cri_tools
     - etcdadm
+  - identifier: eks_anywhere_packages
```

Comment:
let's reorder to put this at the bottom, and add a depend-on field to the packages build
```yaml
depend-on:
- harbor
- hello_eks_anywhere
```

line 205 has an example of adding such a clause, and its indentation.

Reply:
Oh shoot, right, obviously I forgot that part. Thank you.

Reply:
this is because identifiers used in the `depend-on` section need to be defined before being used.

Reply:
Heh, I was kinda wondering if there were ordering requirements.